.github/ISSUE_TEMPLATE/page-translation.yml

```
@@ -0,0 +1,35 @@
+name: 📄 Page request
+title: "page translation: "
+description: Page translation request template
+labels: new command, translation
+body:
+  - type: textarea
+    attributes:
+      label: Command description
+      description: Describe a command you want to be translated
+      placeholder: Tell us what Tl;Dr page you want to see!
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Command details
+      description: Describe any details related to a command
+      placeholder: Tell us a command version for instance
+  - type: dropdown
+    attributes:
+      label: OS
+      description: Specify an operation system this command is implemented for (if you don't know please select "Other")
+      multiple: false
+      options:
+        - Linux
+        - Mac OS
+        - Windows
+        - General
+        - Other
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: OS details
+      description: Describe any details related to an operating system
+      placeholder: Tell us an OS version for instance
```

```suggestion
      placeholder: e.g. OS version
```

Could be worth telling them how to check? 🤔 
For example, on Linux `uname -a` would be great.

I'm betting Windows/macOS have a similar thing.

Might also be out-of-scope for this template though, welcome to ignore this.

---

Maybe it's better to use `lsb_release -a`.

---

Yes, it's a nice idea. 👍🏻